packages/core/src/stylable-processor.ts

```diff
@@ -154,6 +154,9 @@ export const processorWarnings = {
     INVALID_NAMESPACE_REFERENCE() {
         return 'st-namespace-reference dose not have any value';
     },
+    INVALID_NESTING(child: string, parent: string) {
+        return `invalid nesting of ${child} inside ${parent}`;
```

Comment:
```suggestion
        return `Stylable does not support the nesting of rules within rules, found: "${child}" inside "${parent}"`;
```

Reply:
@idoros thoughts about this error message?

Reply:
Currently support?

Reply:
no. but it does parse and find things inside but not transform correctly